### PR TITLE
Mission premable and statement

### PR DESCRIPTION
**Pull request by ryanleesipes**
Let me know how you guys think this looks. Mission preamble and statement.

### CODE REVIEW

File: website/about/index.html [hunk 1]
```diff
@@ -57,6 +57,18 @@ <h3 class="header-section">
...
+          <h4 class="subheader-section">{{ _('Our Mission') }}</h4>
+          <p class="font-md tracking-wide leading-loose mb-6 mt-0 p-links-blue">
+            {{ _('Email has been and continues to be the fundamental communication tool and a significant driver of freedoms on the Internet. But it is also threatened by new non-standards-based, closed and centralized messaging platforms. To prevent this erosion, the public deserves an alternative that is secure and reliable, while  protecting our privacy, enhancing our productivity and serving as a
+            platform for current and future forms of messaging.')) }}
+          </p>
+          <p class="font-md tracking-wide leading-loose mb-6 mt-0 p-links-blue">
+            {{ _('Thunderbird facilitates cross platform, decentralized, open-standard communication, which puts the user in control of their data and workflow. We aspire to offer an interoperable and extensible open-source platform for messaging and managing personal information, such as email, contacts, and appointments.')) }}
+          </p>
+        </article>
+
```
Comment (Alecaddd) [inline website/about/index.html:71]: missing the closing `</aside>` tag.

**Review by Alecaddd — CHANGES_REQUESTED** [review]

**Review by ryanleesipes — COMMENTED** [review]
Ah, good catch @Alecaddd

**Comment by ryanleesipes** [comment]
I don't have the ability to merge the changes, @Sancus or @Alecaddd do you?

File: website/about/index.html [hunk 2]
```diff
@@ -57,6 +57,18 @@ <h3 class="header-section">
...
 
+      <aside class="flex flex-col lg:w-1/2 pl-8 pr-8">
+        <article class="flex flex-col w-full mb-6">
+          <h4 class="subheader-section">{{ _('Our Mission') }}</h4>
+          <p class="font-md tracking-wide leading-loose mb-6 mt-0 p-links-blue">
+            {{ _('Email has been and continues to be the fundamental communication tool and a significant driver of freedoms on the Internet. But it is also threatened by new non-standards-based, closed and centralized messaging platforms. To prevent this erosion, the public deserves an alternative that is secure and reliable, while  protecting our privacy, enhancing our productivity and serving as a
+            platform for current and future forms of messaging.')) }}
+          </p>
+          <p class="font-md tracking-wide leading-loose mb-6 mt-0 p-links-blue">
+            {{ _('Thunderbird facilitates cross platform, decentralized, open-standard communication, which puts the user in control of their data and workflow. We aspire to offer an interoperable and extensible open-source platform for messaging and managing personal information, such as email, contacts, and appointments.')) }}
```
Comment (Sancus) [inline website/about/index.html:68]: extra `)` causes build failure.

File: website/about/index.html [hunk 3]
```diff
@@ -57,6 +57,18 @@ <h3 class="header-section">
         <img class="w-full max-w-4xl mb-10 lg:mb-0 h-auto shadow-lg rounded-sm" src="{{ static('img/thunderbird/team.jpg') }}" alt="Thunderbird Developers Toronto Summit 2014">
       </aside>
 
+      <aside class="flex flex-col lg:w-1/2 pl-8 pr-8">
+        <article class="flex flex-col w-full mb-6">
+          <h4 class="subheader-section">{{ _('Our Mission') }}</h4>
+          <p class="font-md tracking-wide leading-loose mb-6 mt-0 p-links-blue">
+            {{ _('Email has been and continues to be the fundamental communication tool and a significant driver of freedoms on the Internet. But it is also threatened by new non-standards-based, closed and centralized messaging platforms. To prevent this erosion, the public deserves an alternative that is secure and reliable, while  protecting our privacy, enhancing our productivity and serving as a
+            platform for current and future forms of messaging.')) }}
```
Comment (Sancus) [inline website/about/index.html:65]: extra `)` at the end causes build failure.

**Review by Sancus — CHANGES_REQUESTED** [review]
This patch creates a pretty wonky layout with a big white space below the group photo, and misaligned text paragraphs.

image

We probably need to fix that before it can go live.

**Comment by Sancus** [comment]
Completed in #446.

**Pull request closed without merging**